src/ServiceControlInstaller.PowerShell/ServiceControlInstaller.Powershell.dll-help.xml

```diff
@@ -729,10 +775,27 @@ Get-SecurityIdentifier MyDomain\Foo</dev:code>
 	<command:syntax>
 		<command:syntaxItem>
 			<maml:name>Invoke-ServiceControlInstanceUpgrade</maml:name>
+			<command:parameter required="false" variableLength="true" globbing="false" pipelineInput="false" position="named">
+				<maml:name>AuditRetentionPeriod</maml:name>
+				<maml:description>
+					<maml:para>AuditRetention Period controls how long ServiceControl retains messages ingested from the Audit queue in the database.  The value must be a TimeSpan. within the range 1 hour to 365 days. 
+Use the New-TimeSpan cmdlet to generate a TimeSpan valueT</maml:para>
+				</maml:description>
+				<command:parameterValue required="true" variableLength="true">Nullable`1[TimeSpan]</command:parameterValue>
+			</command:parameter>
+			<command:parameter required="false" variableLength="true" globbing="false" pipelineInput="false" position="named">
+				<maml:name>ErrorRetentionPeriod</maml:name>
+				<maml:description>
+					<maml:para>ErrorRetentionPeriod controls how long ServiceControl retains messages ingested from the Error queue in the database.  Only error messages that have been successfully retried or marked as resolved are purged. 
```

Comment:
Is it worth mentioning that the timer starts when the message has been retried or marked as resolved? 

Is it successfully resolved or just retry issued? What about archived? Should we just say "messages that are in the Unresolved state will never be purged"?